go/vt/sqlparser/ast.go

```diff
@@ -3400,6 +3403,29 @@ type ShowTablesOpt struct {
 	AsOf   Expr
 }
 
+// Format formats the node.
+func (node *ShowTablesOpt)Format(buf *TrackedBuffer) {
```

Comment:
Generally we have a parse test that ensures the output is  correct for these things. Want to add one?

Reply:
Good call! Turns out we actually had good parse tests for these forms of `SHOW TABLES` already, but `Show::Format` needed to be changed to delegate to `ShowTablesOpt::Format`. Fixed in 2c69dc10ed9c655af643b7e139de8b927a46a3e5.